### ISSUE TITLE
UMAPI Issue

### ISSUE THREAD

**Issue by jli-alt** [adobe-apiplatform/user-sync.py#664]
Is anyone else having issues syncing new users?  When I try to sync a new user, I'm getting the following error:

2020-12-03 11:31:27 8052 ERROR umapi.action - Error in requestID: action_2 (User: {'user': 'test@mydomain.com', 'requestID': 'action_2'}, Command: {'createFederatedID': {'email': 'test@mydomain.com', 'option': 'ignoreIfAlreadyExists', 'firstname': 'test', 'country': 'US'}}): code: "error.internal.create_failed" message: "Could not create person of type createFederatedID"

I'm running User Sync version: 2.5.0

**Comment by Luci2015** [comment]
It's worth to verify if:
1) inside Admin Console -> Settings -> inside the directory that contains the domain in question -> under the **Sync** tab/menu you have something set up in there. UST will stop working if that menu is not empty (showing "Sync not set up" message and there is an "Add sync" blue button). 

2) the `mydomain.com` is **trusted** in this Org you are synchronizing? If it is **and** if this domain is under an Azure or Google connector sync setup in the main Org (the one that has the claim on this domain), then UST can't just create new FederatedID users in the secondary Org (the one where the domain is trusted). The federatedIDs must exist/be synced in the main Org first, then they can be added to secondary Orgs.

**Comment by jli-alt** [comment]
I just checked my settings.
   
 1.  When I go to the Admin Console -> Settings -> under the Sync tab/menu, this is wheat I see. 
![image](https://user-images.githubusercontent.com/75443011/101086234-93fe4c80-357e-11eb-8eec-59b4e4fada16.png)

      When I click on Go to Settings, this is what I see:
![image](https://user-images.githubusercontent.com/75443011/101087239-f441be00-357f-11eb-9da0-9355d1e3b60c.png)


2. Mydomain.com is trusted in Org I'm syncing.  The domain is not set to with a Azure or Google sync connector sync.

I've had the User Sync tool running for more than 6 months now and I've never encountered this issue before.  This is the first time I'm running into this issue.

**Comment by Luci2015** [comment]
You can use that Remove Sync button and Save, it will remove this unfinished configuration and things will go back to normal.

**Comment by jli-alt** [comment]
Thank you Luci2015!  That resolved the issue.

**Comment by takasead** [comment]
I got the current information of Admin Console sync settings from a customer.
It will be in Japanese, but I will submit the image.
![image](https://user-images.githubusercontent.com/70120450/104546779-4cd0a480-5670-11eb-9c2a-8f04dd7ebe86.png)
In the "Sync not set up" state, I found that the synchronization was not set.

Also, paste a part of the log submitted by the customer. (User name is * 3. Company name is converted to * 4.)
![image](https://user-images.githubusercontent.com/70120450/104547142-29f2c000-5671-11eb-98b3-67019a9f068f.png)

Is there any other solution?
